Tests/FluentAssertions.Specs/FluentAssertions.Specs.csproj

```diff
@@ -55,21 +53,22 @@
   <ItemGroup>
     <PackageReference Include="Chill" Version="4.1.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="Xunit.StaFact" Version="1.0.37" />
     <PackageReference Include="System.Data.DataSetExtensions" Version="4.5.0" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.1">
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.164">
```

Comment:
I upgraded StyleCop.Analyzers 4 lines below.
and the beta.321 fixed the false warnings on `new()`

maybe a wrongly solved merge/rebase conflict?